common/djangoapps/philu_commands/management/commands/auto_generate_certificates_for_open_courses.py

```diff
@@ -67,7 +77,7 @@ class Command(BaseCommand):
 
     def handle(self, *args, **options):
         for course in modulestore().get_courses():
-            if not is_course_valid_for_certificate_auto_generation(course):
+            if not is_course_valid_for_certificate_auto_generation(course) and not has_active_certificate(course):
```

Comment:
Let's please move `not has_active_certificate(course)` call, inside `is_course_valid_for_certificate_auto_generation` method, because actually we have added an other criteria for course valid for certificate auto generation. Then if you mind moving `is_course_valid_for_certificate_auto_generation` method in helpers.py as well that would be great!